server/cmd/start.go

```diff
@@ -86,3 +87,11 @@ func UrlRewrite(r *gin.Engine) gin.HandlerFunc {
 		c.Next()
 	}
 }
+
+func NamespaceStatus(namespaced bool, managedNamespace string) gin.HandlerFunc {
```

Comment:
just wanted to keep the routes in a common file, can do this instead

Reply:
> thought of keeping the routes in a common file, can do this instead. Or instead of setting parameters we can pass them as argument ~ `routes.Routes(router, managedNamespace, namespaced)`. wdyt?

This works too. Or pass the wrapped `SystemInfo`.